src/controllers/usersController.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const usersModel = require('../models/users');
+
+const getAllUsers = (req, res) => {
+  const users = usersModel.getAllUsers();
+
+  return res.status(200).send(users);
+};
+
+const getUserById = (req, res) => {
+  const { userId } = req.params;
+
+  const user = usersModel.getUserById(Number(userId));
```

Comment:
It's better to check if userId is a valid number (not NaN) after converting it using Number().